scripts/validate-config.sh

```
@@ -0,0 +1,181 @@
+    else
+        warn "yq not found, skipping YAML validation for $description"
+        return 0
+    fi
+}
+
+validate_docker_compose() {
+    local files="$1"
+    local description="$2"
+    
+    if command -v docker-compose &> /dev/null; then
+        if docker-compose $files config > /dev/null 2>&1; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)